### PR TITLE
fix: remove network will remove all user annotation

### PR DESCRIPTION
**Pull request by Abirdcfly**
Signed-off-by: Abirdcfly <fp544037857@gmail.com>

### CODE REVIEW

**Comment by Abirdcfly** [comment]
local test pass.

**Review by 0xff-dev — APPROVED** [review]

File: pkg/user/user.go [hunk 1]
```diff
@@ -131,10 +131,17 @@ func ReconcileRemove(u *iam.User, organization, orderer string, idType IDType) (
 	if err != nil {
 		return BlockchainAnnotation{}, errors.Wrap(err, "organization "+organization)
 	}
-	// delete organization from user's annotation
-	err = annotationList.DeleteAnnotation(organization)
-	if err != nil && err != ErrAnnotationNotExist {
-		return BlockchainAnnotation{}, err
+	if orderer != "" {
```
Comment (bjwswang) [inline pkg/user/user.go:134]: should consider  IDType `peer` (same as orderer) in `ReconcileAdd` `ReconcileRemove`

**Comment by bjwswang** [comment]
@Abirdcfly btw... 
- add two samples  `1 orderer cluster ` and `3 orderers cluster`  to `config/samples` 
- update readme

**Comment by Abirdcfly** [comment]
> @Abirdcfly btw...
> 
> * add two samples  `1 orderer cluster ` and `3 orderers cluster`  to `config/samples`
> * update readme

Tracking in a new issue https://github.com/bestchains/fabric-operator/issues/52 , I will submit another pr to fix this issue.

File: pkg/user/user.go (hunk 1 above)
Comment (Abirdcfly) [reply pkg/user/user.go]: Done.

File: pkg/user/user.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ func ReconcileAdd(u *iam.User, organization, orderer string, idType IDType) erro
 	case CLIENT:
 		id = BuildAdminID(u.GetName())
 	case ORDERER:
```
Comment (bjwswang) [inline pkg/user/user.go:86]: should add `case PEER`

Comment (Abirdcfly) [reply pkg/user/user.go:86]: Done

**Review by bjwswang — APPROVED** [review]
LGTM

**Pull request merged**